specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-11-01/keysManagedHsm.json

```diff
@@ -0,0 +1,630 @@
+            "schema": {
+              "$ref": "common.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List key versions in the managed HSM": {
+            "$ref": "./examples/managedHsmListKeyVersions.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ]
+      }
+    }
+  },
+  "definitions": {
+    "ManagedHsmKeyProperties": {
+      "properties": {
+        "attributes": {
+          "$ref": "#/definitions/ManagedHsmKeyAttributes",
+          "description": "The attributes of the key."
+        },
+        "kty": {
+          "type": "string",
+          "minLength": 1,
+          "description": "The type of the key. For valid values, see JsonWebKeyType.",
+          "enum": [
+            "EC",
+            "EC-HSM",
+            "RSA",
+            "RSA-HSM"
+          ],
+          "x-ms-enum": {
+            "name": "JsonWebKeyType",
+            "modelAsString": true
+          }
+        },
+        "keyOps": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "The permitted JSON web key operations of the key. For more information, see JsonWebKeyOperation.",
+            "enum": [
+              "encrypt",
+              "decrypt",
+              "sign",
+              "verify",
+              "wrapKey",
+              "unwrapKey",
+              "import",
+              "release"
+            ],
+            "x-ms-enum": {
+              "name": "JsonWebKeyOperation",
+              "modelAsString": true
+            }
+          }
+        },
+        "keySize": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The key size in bits. For example: 2048, 3072, or 4096 for RSA."
+        },
+        "curveName": {
+          "type": "string",
+          "description": "The elliptic curve name. For valid values, see JsonWebKeyCurveName.",
+          "enum": [
+            "P-256",
+            "P-384",
+            "P-521",
+            "P-256K"
+          ],
+          "x-ms-enum": {
+            "name": "JsonWebKeyCurveName",
+            "modelAsString": true
+          }
+        },
+        "keyUri": {
+          "type": "string",
+          "description": "The URI to retrieve the current version of the key.",
+          "readOnly": true
+        },
+        "keyUriWithVersion": {
+          "type": "string",
+          "description": "The URI to retrieve the specific version of the key.",
+          "readOnly": true
+        },
+        "rotationPolicy": {
+          "$ref": "#/definitions/ManagedHsmRotationPolicy",
+          "description": "Key rotation policy in response. It will be used for both output and input. Omitted if empty"
+        },
+        "release_policy": {
```

Comment:
>release_policy [](http://example.com/codeflow?start=9&length=14)

use camelCase

Reply:
This is a legacy property name used is data plane and we are exposing that existing schema through ARM now so we have to live with this old name to have compatibility with existing resources.